ios/PhyWeb/Backend/PWMetadataRequest.h

```diff
@@ -17,14 +17,15 @@
 #import <Foundation/Foundation.h>
 #import <UriBeacon/UriBeacon.h>
 
-#define METADATA_SERVER_HOSTNAME @"url-caster.appspot.com"
```

Comment:
We should remember to update the README in web-service if this changes names, since we now want to have instructions for how to change the backend.